codegen/src/types.rs

```diff
@@ -170,7 +170,7 @@ impl FieldType {
 
                 if found.is_none() {
                     // recursively search into nested messages
-                    for m in msgs {
+                    for m in msgs.iter() {
```

Comment:
Ah, I got into a habit of being explicit about iterators. But you're quite correct, and I'd like to follow your style.